jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/RetrieveRegistryCredentialsStep.java

```diff
@@ -196,10 +198,12 @@ Authorization retrieveFromCredentialHelper(String credentialHelperSuffix)
     buildLogger.info("Checking credentials from docker-credential-" + credentialHelperSuffix);
 
     try {
-      Authorization authorization =
+      Credentials credentials =
           dockerCredentialHelperFactory
               .newDockerCredentialHelper(registry, credentialHelperSuffix)
               .retrieve();
+      Authorization authorization =
+          Authorizations.withBasicCredentials(credentials.getUsername(), credentials.getPassword());
```

Comment:
Wouldn't `withBasicCredentials(credentials)` would be simpler and match the signature?

Reply:
Currently, we have the `Credentials` in the `configuration` package, whereas `Authorizations` is in the `http` package. I'm not sure if we want `Authorizations` to have a dependency on `Credentials` yet.